AABB_tree/include/CGAL/AABB_triangulation_3_triangle_primitive.h

```diff
@@ -19,60 +19,59 @@
 // Author(s)     : Stéphane Tayeb, Pierre Alliez
 //
 
-#ifndef CGAL_AABB_C3T3_TRIANGLE_PRIMITIVE_H_
-#define CGAL_AABB_C3T3_TRIANGLE_PRIMITIVE_H_
+#ifndef AABB_TRIANGULATION_3_TRIANGLE_PRIMITIVE_H_
+#define AABB_TRIANGULATION_3_TRIANGLE_PRIMITIVE_H_
 
 #include <boost/utility/enable_if.hpp>
 #include <boost/type_traits/is_same.hpp>
 
 namespace CGAL {
     /// \ingroup PkgAABB_tree
-    /// Primitive type that wraps a facet handle of a C3T3,
+    /// Primitive type that wraps a facet handle of a TriangleMesh,
     /// which is used as id, and allows the construction of the datum on
     /// the fly. Since only the facet handle is stored in this primitive,
-    /// the C3T3 from which the AABB tree is built should not be
+    /// the TriangleMesh from which the AABB tree is built should not be
```

Comment:
In those lines 30 and 33, I think `TriangleMesh` should be `CGAL::Triangulation_3`. Because I doubt a `TriangleMesh` has the nested type `Facet`.